buildSrc/src/main/java/baritone/gradle/task/ProguardTask.java

```diff
@@ -195,6 +200,88 @@ private void acquireDependencies() throws Exception {
         }
     }
 
+    // a bunch of bullshit to get the path to the cached jar
+    private Path getMinecraftJar() throws Exception {
+        MappingType mappingType;
+        try {
+            mappingType = getMappingType();
+        } catch (Exception e) {
+            System.err.println("Failed to get mapping type, assuming NOTCH");
+            mappingType = MappingType.SEARGE;
```

Comment:
```
            System.err.println("Failed to get mapping type, assuming NOTCH");
            mappingType = MappingType.SEARGE;
```

this is actually so epic wow
maybe the printout could match the actual mapping type that it sets LOL